### PR TITLE
Update the IAM documentation to be more specific about allowed actions and resources

### PR DESCRIPTION
**Pull request by chrisgilmerproj**
My company doesn't allow me to use actions like `s3:*` and resources like `*` in IAM policy documents. After working out the actions and resources I needed I thought I'd document them back here. I also noticed that the KMS and SNS actions were not listed. KMS is needed to decrypt any objects from encrypted buckets. SNS is required for any SNS topics you publish to.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -147,10 +147,35 @@ following policy document
...
          "Action":[
-            "s3:*"
+            "s3:GetObject",
+            "s3:GetObjectTagging",
+            "s3:PutObjectTagging",
+            "s3:PutObjectVersionTagging",
          ],
          "Effect":"Allow",
-         "Resource":"*"
+         "Resource": ["arn:aws:s3:::<bucket-name-1>/*", "arn:aws:s3:::<bucket-name-2>/*"]
```
Comment (jaygorrell) [inline README.md]:
```suggestion
         "Resource": [
            "arn:aws:s3:::<bucket-name-1>/*", 
            "arn:aws:s3:::<bucket-name-2>/*"
         ]
```

File: README.md [hunk 2]
```diff
@@ -147,10 +147,35 @@ following policy document
...
+      },
+      {
+         "Action":[
+            "s3:GetObject",
+            "s3:GetObjectTagging",
+         ],
+         "Effect":"Allow",
+         "Resource": ["arn:aws:s3:::<av-definition-s3-bucket>/*"]
```
Comment (jaygorrell) [inline README.md]:
```suggestion
         "Resource": [
            "arn:aws:s3:::<av-definition-s3-bucket>/*"
         ]
```

File: README.md [hunk 3]
```diff
@@ -147,10 +147,35 @@ following policy document
...
+      },
+      {
+         "Action":[
+            "kms:Decrypt",
+         ],
+         "Effect":"Allow",
+         "Resource": ["arn:aws:s3:::<bucket-name-1>/*", "arn:aws:s3:::<bucket-name-2>/*"]
```
Comment (jaygorrell) [inline README.md]:
```suggestion
         "Resource": [
            "arn:aws:s3:::<bucket-name-1>/*",
            "arn:aws:s3:::<bucket-name-2>/*"
         ]
```

File: README.md [hunk 4]
```diff
@@ -147,10 +147,35 @@ following policy document
...
+      },
+      {
+         "Action":[
+            "sns:Publish",
+         ],
+         "Effect":"Allow",
+         "Resource": ["arn:aws:sns:::<av-scan-start>", "arn:aws:sns:::<av-status>"]
```
Comment (jaygorrell) [inline README.md]:
```suggestion
         "Resource": [
            "arn:aws:sns:::<av-scan-start>",
            "arn:aws:sns:::<av-status>"
         ]
```

File: README.md (hunk 1 above)
Comment (chrisgilmerproj) [reply README.md]: Fixed in f74f0c3 along with all the others:)

**Comment by chrisgilmerproj** [comment]
This one should be ready to go @jaygorrell ! Thanks for the style points.

**Pull request merged**